Shared/Extensions/UI Extensions/UIImage+Extensions.swift

```diff
@@ -53,25 +54,9 @@ extension UIImage {
     }
 }
 
-extension UIImage: Avatar {
-
-    var givenName: String {
-        return "First"
-    }
-
-    var familyName: String {
-        return "Last"
-    }
-
-    var handle: String {
-        return String()
-    }
```

Comment:
UIImage doesn't really make sense as an Avatar so I just made it an ImageDisplayable.